scripts/installer/ec2-create-instance.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
```

Comment:
It would be slightly more portable to use `#!/usr/bin/env bash` instead of `#!/bin/bash`, although for how the documentation says to use these scripts that won't matter (and in practice, for CentOS and OS X it is in `/bin/`).

Reply:
I'm not sure where I picked up /bin/bash as the path but it always seems to work for me. I'm not aware of when I'd need to use env instead. Maybe if someone installed a newer version of bash? But I'm targeting the system bash.